### ISSUE TITLE
Weird Loss Curve

### ISSUE THREAD

**Issue by HungTeen** [Vegetebird/MHFormer#94]
大佬你好，我用你的代码跑模型结果Loss曲线是这样的，这是正常的吗？为什么测试集的loss和训练集之间差了这么大呢？我还是个刚入门的萌新，希望能够得到你的解答，谢谢。
Excuse me, I ran the model with your code and the loss curve looks like this, is this normal? Why is there such a big difference between the loss of the test set and the training set? I am still a newbie, I hope I can get your reply, thanks.
![image](https://user-images.githubusercontent.com/46000045/219951041-37df8e2b-893b-4daa-8657-fa4061721214.png)

**Comment by Vegetebird** [comment]
It is normal. You can test on Human3.6M with our pretrained model, and you will get the results on MPJPE) in mm​.
